client/src/components/Forms/Interview/InterviewForm.js

```diff
@@ -0,0 +1,215 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Button, Modal, Form, Popconfirm, Steps, message,
+} from 'antd';
+import { QuestionCircleOutlined } from '@ant-design/icons';
+import axios from 'axios';
+import {
+  StepOne, StepTwo, StepThree, StepFour,
+} from './Steps';
+import './InterviewForm.css';
+import StepFive from './Steps/StepFive';
+
+const { Step } = Steps;
+const InterviewForm = () => {
+  const [visible, setVisible] = React.useState(true);
+  const [formData, setFormData] = React.useState({
+    step: 0,
+    specialization: '',
+    language: '',
+    questionCategory: '',
+    time: '',
+    date: '',
+    interviewerId: '',
+  });
+
+  const { step } = formData;
+  const [progressPercent, setProgressPercent] = useState(0);
+  const [availableTime, setAvailableTime] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    if (loading) {
+      const doneBtn = document.getElementById('done');
+      if (doneBtn) doneBtn.disabled = true;
+    } else {
+      const doneBtn = document.getElementById('done') ? document.getElementById('done') : null;
+      if (doneBtn) doneBtn.disabled = false;
+    }
+  }, [loading]);
```

Comment:
Why you are manipulating dom here? you have a state and you can directly disable the btn according to your state.